src/LLVM_Runtime_Linker.cpp

```diff
@@ -888,6 +892,11 @@ std::unique_ptr<llvm::Module> get_initial_module_for_target(Target t, llvm::LLVM
                 user_error << "Metal can only be used on ARM or X86 architectures.\n";
             }
         }
+        if (t.has_feature(Target::D3D12Compute)) {
+            // TODO(marcos) : I think there's more to be done here...
+            modules.push_back(get_initmod_d3d12_abi_patch_64_ll(c));
```

Comment:
Does this only work on 64-bit? Do we need an assertion somewhere that D3D12 is only enabled for a 64-bit target?

Reply:
Unfortunately, yes, at the moment the back-end will only work on 64bit targets.
I need to patch the ABI with some assembly to get it to work on 32bit targets, but that's a bit of a low priority task for me at the moment...
I was not sure about the best way to only have the 64bit back-end modules built; can you give some pointers on what/where to change in order to have the target requirements enforced?